fastlane/lib/fastlane/new_action.rb

```diff
@@ -31,8 +31,16 @@ def self.generate_action(name)
       UI.success "Created new action file '#{path}'. Edit it to implement your custom action."
     end
 
+    def self.check_action_name_from_args(new_action_name)
+      if !new_action_name.nil? && name_valid?(new_action_name)
+        new_action_name
+      else
+        puts "Name invalid, try again!"
```

Comment:
It feels like this message will be printed in the basic case where no name was provided. Can you confirm?